pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[tool.poetry]
+name = "schematicpy"
+version = "0.1.11"
+description = "Package for data model and data ingress management"
+authors = [
+    "Milen Nikolov <milen.nikolov@sagebase.org>",
+    "Sujay Patil <sujay.patil@sagebase.org>",
+    "Bruno Grande <bruno.grande@sagebase.org>",
+    "Xengie Doan <xengie.doan@sagebase.org>"
+]
+
+readme = "README.md"
+
+homepage = "https://github.com/Sage-Bionetworks/schematic"
+repository = "https://github.com/Sage-Bionetworks/schematic"
+documentation = "https://github.com/Sage-Bionetworks/schematic"
+
+keywords = ["schema", "metadata", "validation", "data model", "linked data"]
```

Comment:
Just to note a few more keywords (in addition to the current list):
- schema
- data model
- linked data
- metadata
- validation
- annotation
- knowledge graph/ontology
- ingress

No need to add them (_you can if you want_) in this PR, just noting them for reference.

Reply:
I'm limited to 5 keywords ([source](https://python-poetry.org/docs/pyproject/#keywords)), so I added the additional ones as a comment for future reference.